conf/battle/drops.conf

```diff
@@ -18,20 +18,23 @@ item_first_get_time: 3000
 
 // Grace time during which only the first and second person who did the most damage to a monster can get the item? (in milliseconds)
 // (Takes effect after item_first_get_time elapses)
-item_second_get_time: 1000
+item_second_get_time: 2000
 
 // Grace time during which only the first, second and third person who did the most damage to a monster can get the item? (in milliseconds)
 // (Takes effect after the item_second_get_time elapses)
-item_third_get_time: 1000
+item_third_get_time: 2000
 
-// Grace time to apply to MvP reward items when the Most Valuable Player can't get the prize item and it drops on the ground? (in milliseconds)
+// Grace time during which only the person who did the most damage to a boss can get the item? (in milliseconds)
+// Also applies to MvP reward items when the Most Valuable Player can't get the prize item and it drops on the ground
```

Comment:
```suggestion
// Also applies to MvP reward items, when the Most valuable Player can't get the prize item and it drops on the ground
```

Reply:
Writing the v lower case seems strange to me. Do you really prefer that?

Reply:
I dont care. But keep it consistent. It is different everywhere in this PR

Reply:
It's mixed all along the code, so we better start changing them to uppercase